data/tutorials/getting-started/3_01_ocaml_on_windows.md

```diff
@@ -38,15 +241,6 @@ The guidance is based on the availability table below:
 ╰──────────────────────────────────────────────────────────────────────────────────────────╯
 ```
 
-## opam 2.2
-
-After the [successful release](https://github.com/ocaml/opam/releases/tag/2.1.0)
-of opam 2.1.0, the [next version](https://github.com/ocaml/opam/projects/2) of
-opam will focus on closing the gap to fully support Windows. This includes
-supporting an external dependency installation for Windows and integrating it with the
-Windows shell. From an `opam-repository` perspective, the `ocaml-base-compiler`
-packages will support the MinGW-w64 and MSVC variants.
-
 ## Installation Environments
 
 ### `opam-repository-mingw`
```

Comment:
I think we can remove this section entirely since it is deprecated and opam-repository is now compatible with Windows.